packages/runtime/src/useCases/transport/devices/SetCommunicationLanguage.ts

```diff
@@ -0,0 +1,25 @@
+import { Result } from "@js-soft/ts-utils";
+import { LanguageISO639 } from "@nmshd/content";
+import { DeviceController } from "@nmshd/transport";
+import { Inject } from "typescript-ioc";
+import { RuntimeErrors, UseCase } from "../../common";
+
+export interface SetCommunicationLanguageRequest {
+    communicationLanguage: string;
+}
+
+export class SetCommunicationLanguageUseCase extends UseCase<SetCommunicationLanguageRequest, void> {
+    public constructor(@Inject private readonly deviceController: DeviceController) {
+        super();
+    }
+
+    protected async executeInternal(request: SetCommunicationLanguageRequest): Promise<Result<void>> {
+        if (request.communicationLanguage in LanguageISO639) {
```

Comment:
early return pls

Reply:
AFAIK this is a matter of taste

Reply:
no, it isn't.

Reply:
please see all the other UseCases, all of them first do the validation and then the action. Why should we do it different here because some matter of taste?

Reply:
I didn't look at the other use cases, I look at this and think it's fine :) . It will be gone anyway using the enum